biomes.h

```diff
@@ -21,79 +21,71 @@ class UV {
 constexpr float waterBaseHeight = 64.0f;
 
 enum class BIOME : unsigned char {
-  biOcean = 0,
-  biPlains = 1,
-  biDesert = 2,
-  biExtremeHills = 3,
-  biForest = 4,
-  biTaiga = 5,
-  biSwampland = 6,
-  biRiver = 7,
-  biNether = 8,
-  biEnd = 9,
-  biFrozenOcean = 10,
-  biFrozenRiver = 11,
-  biTundra = 12,
-  biIceMountains = 13,
-  biMushroomIsland = 14,
-  biMushroomShore = 15,
-  biBeach = 16,
-  biDesertHills = 17,
-  biForestHills = 18,
-  biTaigaHills = 19,
-  biExtremeHillsEdge = 20,
-  biJungle = 21,
-  biJungleHills = 22,
-  biJungleEdge = 23,
-  biDeepOcean = 24,
-  biStoneBeach = 25,
-  biColdBeach = 26,
-  biBirchForest = 27,
-  biBirchForestHills = 28,
-  biRoofedForest = 29,
-  biColdTaiga = 30,
-  biColdTaigaHills = 31,
-  biMegaTaiga = 32,
-  biMegaTaigaHills = 33,
-  biExtremeHillsPlus = 34,
-  biSavanna = 35,
-  biSavannaPlateau = 36,
-  biMesa = 37,
-  biMesaPlateauF = 38,
-  biMesaPlateau = 39,
-  biSunflowerPlains = 40,
-  biDesertM = 41,
-  biExtremeHillsM = 42,
-  biFlowerForest = 43,
-  biTaigaM = 44,
-  biSwamplandM = 45,
-  biIcePlainsSpikes = 46,
-  biJungleM = 47,
-  biJungleEdgeM = 48,
-  biBirchForestM = 49,
-  biBirchForestHillsM = 50,
-  biRoofedForestM = 51,
-  biColdTaigaM = 52,
-  biMegaSpruceTaiga = 53,
-  biMegaSpruceTaigaHills = 54,
-  biExtremeHillsPlusM = 55,
-  biSavannaM = 56,
-  biSavannaPlateauM = 57,
-  biMesaBryce = 58,
-  biMesaPlateauFM = 59,
-  biMesaPlateauM = 60,
+  biOcean,
+  biPlains,
+  biDesert,
+  biExtremeHills,
+  biForest,
+  biTaiga,
+  biSwampland,
+  biRiver,
+  biNether,
+  biEnd,
+  biFrozenOcean,
+  biFrozenRiver,
+  biTundra,
+  biIceMountains,
+  biMushroomIsland,
+  biMushroomShore,
+  biBeach,
+  biDesertHills,
+  biForestHills,
+  biTaigaHills,
+  biExtremeHillsEdge,
+  biJungle,
+  biJungleHills,
+  biJungleEdge,
+  biDeepOcean,
+  biStoneBeach,
+  biColdBeach,
+  biBirchForest,
+  biBirchForestHills,
+  biRoofedForest,
+  biColdTaiga,
+  biColdTaigaHills,
+  biMegaTaiga,
+  biMegaTaigaHills,
+  biExtremeHillsPlus,
+  biSavanna,
+  biSavannaPlateau,
+  biMesa,
+  biMesaPlateauF,
+  biMesaPlateau,
+  biSunflowerPlains,
+  biDesertM,
+  biExtremeHillsM,
+  biFlowerForest,
+  biTaigaM,
+  biSwamplandM,
+  biIcePlainsSpikes,
+  biJungleM,
+  biJungleEdgeM,
+  biBirchForestM,
+  biBirchForestHillsM,
+  biRoofedForestM,
+  biColdTaigaM,
+  biMegaSpruceTaiga,
+  biMegaSpruceTaigaHills,
+  biExtremeHillsPlusM,
+  biSavannaM,
+  biSavannaPlateauM,
+  biMesaBryce,
+  biMesaPlateauFM,
+  biMesaPlateauM,
+  biLava,
+  biFlowingRiver,
 
-  teDirt = 61,
-  teStone = 62,
-  
-  waterRiver = 63,
-  waterOcean = 64,
-  waterRiverFrozen = 65,
-  waterOceanFrozen = 66,
-
-  liLava = 67,
-
-  NUM_BIOMES = 68,
+  NUM_BIOMES,
```

Comment:
Thanks for removing the numbers :).  I see what you're doing here with NUM_BIOMES, and I think that's fine, but please leave a comment to indicate that NUM_BIOMES needs to remain the last entry in the enum.